distributed-calculator/react-calculator/server.js

```diff
@@ -14,41 +18,92 @@ const daprUrl = `http://localhost:${daprPort}/v1.0/invoke`;
 const stateStoreName = `statestore`;
 const stateUrl = `http://localhost:${daprPort}/v1.0/state/${stateStoreName}`;
 
+const debugMode = process.env.DEBUG_MODE || false; // Emit debug level logs for State management
+
 /**
 The following routes forward requests (using pipe) from our React client to our dapr-enabled services. Our Dapr sidecar lives on localhost:<daprPort>. We invoke other Dapr enabled services by calling /v1.0/invoke/<DAPR_ID>/method/<SERVICE'S_ROUTE>.
 */
 
 app.post('/calculate/add', async (req, res) => {
-  const addUrl = `${daprUrl}/addapp/method/add`;
-  req.pipe(request(addUrl)).pipe(res);
+  const appUrl = `${daprUrl}/addapp/method/add`;
+
+  // Dapr invoke add app
```

Comment:
is there a need for comments here?

Reply:
I think we comment just the key dapr concept, and explain why casting.  Yes.  This took me a long time to figure out.